conda/recipes/versions.yaml

```
@@ -155,7 +155,7 @@ s3fs_version:
 scikit_build_version:
   - '=0.13.1'
 scikit_image_version:
-  - '>=0.19.0,<0.21.0'
+  - '>=0.19.0,<1.0'
```

Should be `<1.0a0` to prevent prereleases from being picked up.
```suggestion
  - '>=0.19.0,<1.0a0'
```

---

Thanks!